conf/views/disclosure/disclosureMarketable.njk

```diff
@@ -0,0 +1,64 @@
+{% extends "includes/layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"     import govukBackLink %}
+{% from "govuk/components/radios/macro.njk"        import govukRadios %}
+{% from "govuk/components/button/macro.njk"        import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "macros/title.njk"                         import title %}
+
+{% block pageTitle %}
+  {{ title(messages("disclosureMarketable.title"), form.errors) }}
+{% endblock %}
+
+{% block mainContent %}
+
+  {{ govukBackLink({
+    text: messages("site.back")
+  }) }}
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        {% if form.errors.length > 0 %}
+          {{ govukErrorSummary({
+            "titleText": messages("error.summary.title"),
+            "errorList": form.errors
+          }) }}
+        {% endif %}
+
+        <form action="{{ routes.controllers.disclosure.DisclosureMarketableController.onSubmit(mode).url }}" method="post">
+        <fieldset class="govuk-fieldset">
+         {{ csrf() | safe }}
+
+        <legend>
+            <h1 class="govuk-heading-xl" id="heading">
+                {{ messages("disclosureMarketable.heading")}}
+            </h1>
+        </legend>
+
```

Comment:
Opening fieldset tag is in the wrong place and h1 doesn't need to be inside a legend.